js/contact.js

```diff
@@ -77,26 +77,40 @@ function deselect(contactLink) {
 
 function displayContact(contact) {
 
-	document.getElementById("info-fullname").innerHTML = contact.FirstName + " " + contact.LastName;
-	document.getElementById("info-initials").innerHTML = contact.FirstName[0] + contact["LastName"][0];
-	document.getElementById("info-first-name").innerHTML = contact["FirstName"];
-	document.getElementById("info-last-name").innerHTML = contact["LastName"];
-	document.getElementById("info-phone-number").innerHTML = contact["PhoneNumber"];
-	document.getElementById("info-address").innerHTML = contact["Address"];
-	document.getElementById("info-zip").innerHTML = contact["ZIP"];
-	document.getElementById("info-city").innerHTML = contact["City"];
-	document.getElementById("info-state").innerHTML = contact["State"];
-
-	// Get editing ready
```

Comment:
Although I'm a little shady on what some of this does, it doesn't seem like the new loop manages the edit pane's placeholders.

Reply:
Good catch! I'll go ahead and add that back

Reply:
Once this is done go ahead and push it and I'll merge. This whole update really brings a layer of professionalism to the site, I'm loving it :D